.gitignore

```diff
@@ -8,3 +8,5 @@ report.yml
 report.json
 tern.log
 cache.yml
+vagrant/.*
+.idea
```

Comment:
I am not familiar with the .idea file/folder. Is it required for vagrant to work?

Reply:
Hey @nishakm , .idea folder is IDE specific folder created by intellij IDEs to store project related stuff.

Reply:
@xtreme-vikram-yadav since this is IDE specific, could you leave it out of the .gitignore?

Reply:
@nishakm it's a minor thing but this normally shouldn't be committed to this project and will make it convenient for contributors to look at tracked files

Reply:
@nishakm Update should be in

Reply:
@xtreme-vikram-yadav Thank you! This looks good.